src/core/controller.ts

```diff
@@ -221,10 +221,10 @@ export class Controller {
       answer = await this.executeRuleTypeExecutors(rule, message);
     } catch (e) {
       let expectedError = false;
-      if (e instanceof ValidatorError) {
+      if (e instanceof bot.exceptions.ValidatorError) {
```

Comment:
remove the imports used on top of the file
https://github.com/rafaelverger/yve-bot/blob/dfa576f9dfe051490ae54d2b05371c3c03e4cf1f/src/core/controller.ts#L3

and change all occurrences to use `bot.exceptions.X`

Reply:
nice!